poo/abstraction/en/Abstraction.md

```diff
@@ -0,0 +1,117 @@
+# Abstraction
+
+The **abstraction** idea, is chopper a class in parts, to be able to share (via a parameter passed in method for example) **an only** of this part rather the object “in full”.

```

Comment:
Not sure about the `chopper` translation. Maybe `divide` or `cut out` are better translations.
Also I think `only one` is a better translation than `an only`.

Reply:
Tanks, the changes have been made!